src/modules/displayList.js

```diff
@@ -0,0 +1,94 @@
+import { listItems, addTask } from './const.js';
+import { completedTrue, completedFalse } from './checkbox.js';
+
+let listArr = [];
+
+const addList = () => {
+  const obj = {};
+  obj.index = listArr.length + 1;
+  obj.description = addTask.value;
+  obj.completed = false;
+  listArr.push(obj);
+};
+
+const pushToLocal = () => {
+  localStorage.setItem('listArr', JSON.stringify(listArr));
+};
+
+const pushList = () => {
+  listItems.innerHTML = '';
+  listArr.forEach((obj) => {
+    const toDo = `<li>
+      <div><input class= "check-box" type="checkbox" data-id="${obj.index}"></div>
+      <div class="title a-list">
+          <input data-id="${obj.index}" class= "list-input" type = "text" value = "${obj.description}">
+          <button class="delete-btn" data-id="${obj.index}"><i class="fa fa-ellipsis-v"></i></button>
+      </div>
+    </li>`;
+    listItems.innerHTML += toDo;
+    addTask.value = '';
+  });
+  const listRemoveBtn = document.querySelectorAll('.delete-btn');
+
+  listRemoveBtn.forEach((button) => {
+    button.addEventListener('click', () => {
+      const dataSet = parseInt(button.dataset.id, 10);
+      const buttonId = listArr.findIndex((object) => object.index === dataSet);
+      const deleted = (id) => {
+        listArr.splice(id, 1);
+        for (let i = 0; i < listArr.length; i += 1) {
+          listArr[i].index = i + 1;
+        }
+        pushList();
+        localStorage.setItem('listArr', JSON.stringify(listArr));
```

Comment:
Please use the pushToLocal function on line 14 instead of repeating the code all over.